Framework/API/inc/MantidAPI/FileFinder.h

```diff
@@ -49,10 +50,11 @@ class MANTID_API_DLL FileFinderImpl {
   std::vector<std::string> findRuns(const std::string &hintstr, const std::vector<std::string> &exts = {},
                                     const bool useExtsOnly = false) const;
   /// DO NOT USE! MADE PUBLIC FOR TESTING ONLY.
-  const Kernel::InstrumentInfo getInstrument(const std::string &hint) const;
+  const Kernel::InstrumentInfo getInstrument(const std::string &hint, const bool &throwOnThrow = false) const;
```

Comment:
They do have a big warning saying not to use this method, but I believe they meant for it not to be used instead of `ConfigService::Instance().getInstrument()`, which is the general one. I am using this method in `ISISInstrDataCache.cpp` which is only used in `FileFinder.cpp` so I think it's okay for me to use it?